### PR TITLE
Update FlatList Optimization Guide for FCs

### PR DESCRIPTION
**Pull request by Azelisi**
My fix solves the problem Update FlatList Optimization Guide for FCs #3677

### CODE REVIEW

**Comment by Huinko** [comment]
Replaced Use shouldComponent Update with Use memo

componentUpdate
memo

File: docs/optimizing-flatlist-configuration.md [hunk 1]
```diff
@@ -93,16 +93,30 @@ The more complex your components are, the slower they will render. Try to avoid
 
 The heavier your components are, the slower they render. Avoid heavy images (use a cropped version or thumbnail for list items, as small as possible). Talk to your design team, use as little effects and interactions and information as possible in your list. Show them in your item's detail.
 
-### Use shouldComponentUpdate
+### Use Memo()
```
Comment (NickGerleman) [inline docs/optimizing-flatlist-configuration.md:96]:
```suggestion
### Use `memo()`
```

**Review by NickGerleman — APPROVED** [review]
This is great!

**Comment by Simek** [comment]
@NickGerleman Can we merge this in? 🙂

**Pull request merged**